tests/assertions.js

```diff
@@ -182,3 +183,36 @@ export function itRejectsInvalidCryptosuite(expectedValidSuites, {
     await verificationFail({credential, verifier: endpoint});
   });
 }
+
+export async function shouldBeBaseProofValue({proof, name}) {
+  expect(
+    proof,
+    `Expected VC from issuer ${name} to have an ' +
+    '"ecdsa-sd-2023" proof`).to.exist;
+  expect(
+    proof.proofValue,
+    `Expected VC from issuer ${name} to have a ' +
+    '"proof.proofValue"`
+  ).to.exist;
+  expect(
+    proof.proofValue,
+    `Expected VC "proof.proofValue" from issuer ${name} to be ` +
+    'a string.'
+  ).to.be.a.string;
+  //Ensure the proofValue string starts with u, indicating that it
+  //is a multibase-base64url-no-pad-encoded value, throwing an
+  //error if it does not.
+  expect(
+    proof.proofValue.startsWith('u'),
+    `Expected "proof.proofValue" to start with u received ` +
+    `${proof.proofValue[0]}`).to.be.true;
+  // now test the encoding which is bs64 url no pad for this suite
+  expect(
+    shouldBeBs64UrlNoPad(proof.proofValue),
+    'Expected "proof.proofValue" to be bs64 url no pad encoded.'
```

Comment:
When aimed at humans, best to use the full string for "bs64 url no pad encoded".